sw/device/tests/dif/dif_entropy_smoketest.c

```diff
@@ -51,14 +51,15 @@ bool test_main() {
   CHECK(dif_entropy_configure(&entropy, config) == kDifEntropyOk);
 
   uint32_t entropy_data[kEntropyDataNumWords];
+  uint32_t result = 0;
   for (uint32_t i = 0; i < kEntropyDataNumWords; ++i) {
     // wait for entropy to become available
     while (dif_entropy_read(&entropy, &entropy_data[i]) != kDifEntropyOk)
       ;
```

Comment:
Not your code, but can you replace this with braces instead?

Reply:
I will let @moidx comment on this.